artifacts/definitions/Server/Utils/CreateCollector.yaml

```diff
@@ -386,11 +391,11 @@ sources:
       )
 
       LET Target <= tool_name[0].Type
-
+      
       // This is what we will call it.
-      LET CollectorName <= format(
-          format='Collector_%v',
-          args=inventory_get(tool=Target).Definition.filename)
+      LET CollectorName <= if(condition=opt_collector_filename, 
```

Comment:
This can be 

```sql
LET CollectorName <= opt_collector_filename || 
     format(format='Collector_%v', args=inventory_get(tool=Target).Definition.filename)
```